frontend/app/routes/application.js

```diff
@@ -1,15 +1,37 @@
 import { inject as service } from '@ember/service';
 import Route from '@ember/routing/route';
 import DS from 'ember-data';
-import ApplicationRouteMixin from 'ember-simple-auth/mixins/application-route-mixin';
 import { UnauthorizedError, ForbiddenError } from 'ember-ajax/errors';
 
-export default Route.extend(ApplicationRouteMixin, {
-  session: service(),
+export default Route.extend({
+  session: service('keycloak-session'),
   moment: service(),
 
   beforeModel() {
     this.get('moment').setLocale('de');
+    this._super(...arguments);
```

Comment:
`this._super(...arguments)` ist hier glaube ich nicht nötig

Reply:
Ist so aus der Package Doku kopiert 😁 werde noch schauen ob es nötig ist.